v2.0/enable-node-map.md

```diff
@@ -1,7 +1,200 @@
+
+The Localities debug page displays the following:
+
+- Localities configuration that you set up while starting the nodes with the `--locality` flags.
+- Nodes corresponding to each locality.
+- Latitude and longitude coordinates for each locality/node.
+
+On the page, ensure that every node has a locality as well as latitude/longitude coordinates assigned to them.
+
+### Unable to Assign Latitude/Longitude Coordinates to Localities
+
+You won't be able to assign latitude/longitude coordinates to localities if the components of your localities have the same name. For example, consider the following partial configuration:
+
+|  Node | Region | Datacenter |
+|  ------ | ------ | ------ |
+|  Node1 | us-east | datacenter-1 |
+|  Node2 | us-west | datacenter-1 |
+
+In this case, if you try to set the latitude/longitude coordinates to the datacenter level of the localities, you will get the "primary key exists" error and the **Node Map** won't be displayed. You can, however, set the latitude/longitude coordinates to the region components of the localities, and the **Node Map** will be displayed.
+
+### Node Map Not Displayed for All Locality Levels
+
+The **Node Map** is displayed only for the locality levels that have latitude/longitude coordinates assigned to them: 
+
+- If you assign the latitude/longitude coordinates at the region level, the **Node Map** shows the regions on the world map. However, when you drill down to the datacenter and further to the individual nodes, the world map disappears and the datacenters/nodes are plotted in a circular layout.  
+- If you assign the latitude/longitude coordinates at the datacenter level, the **Node Map** shows the regions with single datacenters at the same location assigned to the datacenter, while regions with multiple datacenters are shown at the center of the datacenter coordinates in the region. When you drill down to the datacenter levels, the **Node Map** shows the datacenter at their assigned coordinates. Further drilling down to individual nodes shows the nodes in a circular layout. 
+
+[Assign latitude/longitude coordinates](#step-4-set-the-latitudes-and-longitudes-for-the-localities) at the locality level that you want to view on the **Node Map**.
+
+### **Capacity Used** Value Displayed is More Than Configured Capacity
+
+If you are running multiple nodes on a single machine (not recommended), the **Capacity Used** value may be incorrect. This is because when multiple nodes are running on a single machine, the machine's hard disk is treated as separate stores for each node, while in reality, only one hard disk is used for all nodes. The **Capacity Used** value is then displayed as the hard disk capacity used multiplied by the number of nodes on the machine.
```

Comment:
Rich created an include with this message. Let's use it here: https://github.com/cockroachdb/docs/blob/master/_includes/available-capacity-metric.md

However, I'd like you to make a few minor changes to the include content to make it work in all contexts. 

Instead of:

> the Capacity Used metric in the Summary panel and the available capacity shown in the Capacity graph in the Admin UI are incorrect.

Please use:
> capacity metrics in the Admin UI are incorrect.

Reply:
Done.